src/mpsc/blocking.rs

```diff
@@ -351,6 +351,124 @@ feature! {
             }
         }
 
+        /// Reserves a slot in the channel to mutate in place, blocking until
+        /// there is a free slot to write to, waiting for at most `timeout`.
+        ///
+        /// This is similar to the [`send`] method, but, rather than taking a
```

Comment:
nit: maybe we should reference `send_timeout` here instead?
```suggestion
        /// This is similar to the [`send_timeout`] method, but, rather than taking a
```